pegasus/sites.v3/code.org/public/custom-certificates.haml

```diff
@@ -70,6 +72,6 @@ theme: none
       -unless count == 0
         .page-break
       -name.chomp!
-      %img{:src=>"#{certificate_image_url(name:name, sponsor:sponsor, course:course, course_title:course_name)}", :width=>"100%"}
+      %img{:src=>"#{certificate_image_url(name:name, sponsor:sponsor, course:script, course_title:script_name)}", :width=>"100%"}
```

Comment:
one example of where I didn't change stuff in this PR - the name of the params in `certificate_image_url` are course/course_title